Classes/ErrorHandler.php

```diff
@@ -65,12 +72,38 @@ public function handleException($exception, array $extraData = [])
             $extraData['referenceCode'] = $exception->getReferenceCode();
         }
 
-        $this->client->captureException($exception, [
-                'message' => $exception->getMessage(),
-                'extra' => $extraData,
-                'tags' => $tags
+        $data = [
+            'message' => $exception->getMessage(),
+            'extra' => $extraData,
+            'tags' => $tags,
+        ];
+        $data = array_merge_recursive($data, $this->getBrowserContext(), $this->getOsContext());
+
+        $this->client->captureException($exception, $data);
+    }
+
+    protected function getBrowserContext(): array
+    {
+        return [
+            'contexts' => [
```

Comment:
The supported values are documented here
https://docs.sentry.io/development/sdk-dev/interfaces/contexts/